src/getrandom.rs

```diff
@@ -1,9 +1,42 @@
 //! Implementation using libc::getrandom
+//!
+//! Available since:
+//!   - Linux Kernel 3.17, Glibc 2.25, Musl 1.1.20
+//!   - Android API level 23 (Marshmallow)
+//!   - NetBSD 10.0
+//!   - FreeBSD 12.0
+//!   - Solaris 11.3
+//!   - Illumos since Dec 2018
+//!   - DragonFly 5.7
+//!   - Hurd Glibc 2.31
+//!   - shim-3ds since Feb 2022
+//!
+//! For all platforms, we use the default randomness source (the one used
+//! by /dev/urandom) rather than the /dev/random (GRND_RANDOM) source.
+//!   - On Linux, "/dev/urandom is preferred and sufficient in all use cases".
+//!   - On NetBSD, "there is no reason to ever use [GRND_RANDOM]".
```

Comment:
`[GRND_RANDOM]` is a dangling link. Also it's worth to add source links for the quotes.

Reply:
Fixed the link issue. The additional quotes are from the man pages linked in `lib.rs`, so I just updated the comment to "see the linked man pages in lib.rs".